### PR TITLE
Handle pre-populated vat_amount

### PR DESCRIPTION
**Pull request by doctorpod**
PT: https://www.pivotaltracker.com/story/show/118554905

### CODE REVIEW

File: app/models/claim/base_claim.rb [hunk 1]
```diff
@@ -124,7 +124,22 @@ class BaseClaim < ActiveRecord::Base
...
+          atts["expense_type_id"].blank? &&
+          atts["location"].blank? &&
+          atts["distance"].blank? &&
+          atts["reason_id"].blank? &&
+          atts["reason_text"].blank? &&
+          atts["hours"].blank? &&
+          atts["amount"].blank?
+      },
+      allow_destroy: true
     accepts_nested_attributes_for :disbursements,     reject_if: :all_blank, allow_destroy: true
```
Comment (stephenrichards) [inline app/models/claim/base_claim.rb:128]: I think we need tests for this - I suspect that if we fill in a value in the vat amount and none of the other fields, the record will not be created.
